app/src/main/java/org/wikipedia/dataclient/Service.kt

```diff
@@ -59,6 +59,18 @@ interface Service {
         @Query("gsroffset") gsrOffset: String?
     ): Observable<MwQueryResponse>
 
+    @GET(
+        MW_API_PREFIX + "action=query&redirects=" +
+                "&converttitles=&prop=description|info" +
+                "&generator=prefixsearch&gpsnamespace=0&list=search&srnamespace=0" +
+                "&inprop=varianttitles" +
+                "&srwhat=text&srinfo=suggestion&srprop=&sroffset=0&srlimit=1"
+    )
+    suspend fun searchSubPages(@Query("gpssearch") searchTerm: String?,
```

Comment:
A trailing slash is missing. For example, https://de.wikipedia.org/w/api.php?action=query&list=prefixsearch&pssearch=Diskussion:Gl%C3%A4ubiger returns `Diskussion:Gläubigerschutz` as well, even though it’s definitely not an archive of [Diskussion:Gläubiger](https://de.wikipedia.org/wiki/Diskussion:Gl%C3%A4ubiger). https://de.wikipedia.org/w/api.php?action=query&list=prefixsearch&pssearch=Diskussion:Gl%C3%A4ubiger/ returns the expected result (nothing, as the given talk page has no archives).

Reply:
Already handled in #3475.